ckanext/switzerland/templates/package/base.html

```diff
@@ -4,7 +4,7 @@
   {% if pkg %}
     {% set dataset = h.dataset_display_name(pkg) %}
     {% if pkg.organization %}
-      {% set organization = h.get_translated(pkg.organization, 'title') or pkg.organization.name %}
+      {% set organization = h.parse_and_localize(pkg.organization.title) %}
```

Comment:
have you considered leaving in the fallback to `organization.name`?

Reply:
Thanks, I'll add that back in. :)